tests/smoketest.py

```diff
@@ -62,6 +71,13 @@ def run(command, **kwargs):
 run("jq -n '.es_query.query.match[env.k]=env.v' | http --check-status https://${API_HOST}/v1/search > res.json",
     env=dict(os.environ, k="files.sample_json.uuid", v=sample_id))
 
+for replica in "aws", "gcp":
+    res = run(f"hca put-subscriptions --callback-url https://example.com/ --query '{{}}' --replica {replica}",
+              runner=check_output)
+    sub_id = json.loads(res.decode())["uuid"]
```

Comment:
We should also uploaded a file and check subscriber is notified.

Reply:
maybe just change the order of the tests to create a subscription earlier and deleted it later.

Reply:
That's outside the scope of this PR. Let's address that in a separate PR (which will be much more complicated, since checking that a subscriber is notified will require either standing up an HTTPS test endpoint, or at least looking at server logs). Can you please approve this one so I can merge?

Reply:
As we talk about, modify the --query '{{}}' to a valid query.

Reply:
`{}` (which is what that f-string compiles to) is a (syntactically) valid query, so the test passes.

We should probably make it more semantically meaningful in the future, in a separate PR.